src/firstSuggestionDecoration.ts

```diff
@@ -0,0 +1,74 @@
+import { DecorationOptions, DecorationRangeBehavior, ExtensionContext, Position, Range, TextDocument, TextEditor, window } from "vscode";
+import getHintColor from "./inlineSuggestions/hintColor";
+import { firstSuggestionDecorationAlreadyDisplayed, setFirstSuggestionFlag } from './firstSuggestionDecorationFlag';
+import { fireEvent, getState } from "./binary/requests/requests";
+import isInTheLastHour from "./utils/time.utils";
+import { getTabnineExtensionContext } from './globals/tabnineExtensionContext';
+
+// Anchor hint to the right of the line
+const firstSuggestionDecorationType = window.createTextEditorDecorationType({
+    rangeBehavior: DecorationRangeBehavior.ClosedOpen,
+});
+const maxSmallIntegerV8 = 2 ** 30; // Max number that can be stored in V8's smis (small integers)
```

Comment:
we've got a `consts` file somewhere, i think it can be moved to there (with all caps)